patient-appointment-api.yml

```diff
@@ -70,7 +112,7 @@ paths:
         \ result in the termination of information exchange.\n\n## Overview\nManage\
         \ a patient's registration on the Patient Panel for ADT monitoring. If an\
         \ ADT notification event occurs, the patient will be processed."
-      operationId: processAdtPanelPatient
+      operationId: initiateAdtPanelPatientRegistration
```

Comment:
I doubt it matters that some IDs are changing, but just mentioning it

Reply:
Reverted this change by putting operationId values directly in the source.